clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -252,7 +252,7 @@ class ClioClient(webClient: ClioWebClient, ioUtil: IoUtil)(
       _ <- checkRemainingArgs(args.remaining)
     } yield {
       val executor = command match {
-        case deliverCommand: DeliverWgsCram  => new DeliverWgsCramExecutor(deliverCommand)
+        case deliverCommand: DeliverCram     => new DeliverCramExecutor(deliverCommand)
```

Comment:
Hrm, is this match non-exhaustive? I don't see a supertype for `DeliverWgsCram` and `DeliverCram`, so I don't think the former will match anything.